airflow-core/src/airflow/api/common/delete_dag.py

```diff
@@ -63,10 +64,20 @@ def delete_dag(dag_id: str, keep_records_in_log: bool = True, session: Session =
     if dag is None:
         raise DagNotFound(f"Dag id {dag_id} not found")
 
-    count = 0
+    """
+    To ensure the TaskInstance and DagRun model is deleted before
+    each of the model DagVersion and BackFill respectively.
+    """
```

Comment:
Nit:
Most of the time this is reserved for docstring. For simple multilines comment we favor this
```suggestion
    # To ensure the TaskInstance and DagRun model is deleted before
    # each of the model DagVersion and BackFill respectively.
```

Reply:
Thank you I updated the docstring styled comment into multiline comment